qa/tasks/cephfs/filesystem.py

```diff
@@ -272,8 +272,14 @@ def get_nonnumeric_values(value):
                      "-Infinity": -float("inf")}
                 return c[value]
 
-            j = json.loads(response_data.replace('inf', 'Infinity'),
-                           parse_constant=get_nonnumeric_values)
+            
+            j = {}
+            try:
+                j = json.loads(response_data.replace('inf', 'Infinity'),
+                            parse_constant=get_nonnumeric_values)
+            except json.decoder.JSONDecodeError:
+                raise RuntimeError(response_data)
```

Comment:
```suggestion
                raise RuntimeError(response_data) # assume it is an error message, pass it up
```

Reply:
If you're going to do this though, you should pass the unmodified `response_data` rather than the one we've already mutated a few times in preparation for the call to `json.dumps`.

Reply:
I don't think we are modifying the `response_data` value after extracting it from `proc`. As `replace()` in python creates a copy of the data, doesn't change the original content.